app/views/LocationShare/LiveLocationStatusBar.tsx

```
@@ -0,0 +1,161 @@
+import React, { useState, useEffect, useRef } from 'react';
+import {
+	View,
+	Text,
+	TouchableOpacity,
+	StyleSheet,
+	Animated,
+	InteractionManager,
+	Linking,
+	Platform,
+	StatusBar
+} from 'react-native';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)